index.js

```diff
@@ -0,0 +1,21 @@
+module.exports.makeArrayLowercase = stringArray => {
+  const lowered = stringArray.map(item=>item.toLowerCase())  
+  return lowered
+}

```

Comment:
can simplify to single line since not reusing `lowered` 
```js
return stringArray.map(item=>item.toLowerCase())  
```

Reply:
It was really necesary to create that much commits??

Reply:
Will do, thanks!